frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -237,18 +237,27 @@ public int getNumberOfGpu() {
         return getIntProperty(MMS_NUMBER_OF_GPU, 0);
     }
 
+    public Properties getConfiguration() {
+        return new Properties(prop);
```

Comment:
Look unmodifiable collections.

Reply:
It seems more efficient to just send back a copy of properties than to convert this to HashMap and unmodifiable map from java.util